src/chainparams.cpp

```diff
@@ -7,6 +7,8 @@
 
 #include <chainparamsseeds.h>
 #include <consensus/merkle.h>
+#include <hash.h> // for signet block challenge hash
+#include <signet.h>
```

Comment:
I think you are not using this one in this file.

Reply:
I do; I reference `g_signet_blocks`, for example.

Reply:
I think it's only for that, right? Perhaps a small reason to put the global here instead of there.

Reply:
Belated, but I am removing this and adding an `extern CScript g_signet_blockscript` in the signet chainparams class.